Rivet/Functions/Export-Migration.ps1

```diff
@@ -971,9 +971,17 @@ where
 
             if( $externalDependencies.ContainsKey($object.object_id) )
             {
-                Write-Warning -Message ('Unable to export {0} {1}: it depends on external object {2}.' -f $object.type_desc,$object.full_name,$externalDependencies[$object.object_id])
-                $exportedObjects[$object.object_id] = $true
-                continue
+                $indexOfReferencedDatabase = [array]::IndexOf($settings.Databases.Name, $externalDependencies[$object.object_id].DatabaseName)

```

Comment:
This should only work if `Get-RivetConfig` orders the `Databases` array based on the value of the `DatabasesOrder` configuration property, which it currently isn't. You'll want to check why tests are passing when the code is still ordering databases by file system ordering.

Reply:
Get-RivetConfig does order it based off of DatabaseOrder. It gets the order from the rivet.json on line #273 and then it uses that order on lines #301-#309 linked below.

https://github.com/webmd-health-services/Rivet/blob/feature/export-migration-db-dependency-order/Rivet/Functions/Get-RivetConfig.ps1#L301-L309

So currently if Get-RivetConfig is called without specifying a database it will use the DatabaseOrder from the rivet.json if it exists(Lines 301 -307), if not it will order it by file system ordering (Line 309).

Would you still want to rework this logic?